content/docs/working-groups/diagnostics/_index.md

```diff
@@ -44,3 +44,8 @@ Not yet.
 
 ## Do I need to attend any meetings?
 We don't have any meetings yet.
+
+# Notes and FAQs
+
+- [Notes]({{< relref "/docs/working-groups/diagnostics/NOTES" >}})
```

Comment:
Can we avoid having sections with short lists of links by using the menu on the sidebar?

Reply:
Each working group already contained a link to their meeting notes, so another link here is somewhat redundant. Perhaps the FAQ link could be added next to (and in the same style as) the existing meeting notes link.

Reply:
David some of the working groups do not contain a list to their meeting notes (an example of this is the Traits working group), would you like us to replicate the way it is embedded in the other working groups?

Reply:
I think it would be fine to add a link to their notes. I believe the traits working group was created before we made the template that most of the working group pages now follow.